### PR TITLE
Create Front skeleton

### LINKED ISSUE #4: [INIT] Begin frontend pages
1. Create endpoints for different screens
2. Create test for the different components

### PR DESCRIPTION
**Pull request by OverCry**
This PR contains code of a basic skeleton for user sign in and log in. Currently, they do not interact with any backend and is waiting on backend support.

This PR also contains code for generating and provided the room code of the user. This is also currently depending on backend endpoints

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,24 +1,122 @@
...
+
+  // boolean for showing signup
+  let [register, setSignup] = useState(false);
+
+  const login = (e) => {
+    e.preventDefault();
+    console.log("check login");
+    console.log(user);
+    console.log(password);
+
```
Comment (Dhors) [inline frontend/src/App.js:28]: Nice

File: frontend/src/Components/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+const SignUp = ({ cancel }) => {
+  let [user, setUser] = useState(undefined);
+  let [password, setPassword] = useState(undefined);
+  let [passwordC, setPasswordC] = useState(undefined);
+
+  const onSubmit = (e) => {
+    e.preventDefault();
+    console.log("Creating account");
+    // // check if passwords are the same
```
Comment (Dhors) [inline frontend/src/Components/SignUp.js:16]: Maybe we could add some username checks

Probably just regex strings

**Review by Dhors — APPROVED** [review]
Nice changes!

File: frontend/src/Components/SignUp.js (hunk 2 above)
Comment (OverCry) [reply frontend/src/Components/SignUp.js:16]: Yeah we can do that when we have an endpoint to go to.

**Pull request merged**